backend/eks/deploy.sh

```diff
@@ -14,6 +14,8 @@ if ! aws sts get-caller-identity ; then
     aws configure
 fi
 
+MAIN="${PWD}"
+
 git clone https://github.com/SUSE/cap-terraform.git
 pushd cap-terraform/eks || exit
```

Comment:
I don't see it using `sr/s3-tf-backend` branch. Is this intentional? I thought `master` don't have required code to handle s3 backend. I might be wrong though.

Reply:
If you look into the s3 bukcet `cap-ci-tf` you will see test tf state saved there. The branch was not needed.